tfimm/train/optimizers/lr_schedules.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any
+from dataclasses import dataclass
+from abc import ABC, abstractclassmethod
+
+import tensorflow as tf
+from tensorflow.keras.optimizers.schedules import LearningRateSchedule
+
+from ..registry import cfg_serializable, get_class
+
+
+class BaseLrSchedule(ABC):
```

Comment:
Can I suggest using `LR` instead of `Lr` in the code. Given that it is an abbreviation I would suggest capitalising both. I am also using, e.g., `MLP` elsewhere in the code.